MediaBrowser.Api/Playback/MediaInfoService.cs

```diff
@@ -242,9 +269,8 @@ private T Clone<T>(T obj)
         {
             // Since we're going to be setting properties on MediaSourceInfos that come out of _mediaSourceManager, we should clone it
             // Should we move this directly into MediaSourceManager?
-
-            var json = _json.SerializeToString(obj);
-            return _json.DeserializeFromString<T>(json);
+            var json = JsonSerializer.SerializeToUtf8Bytes(obj);
+            return JsonSerializer.Deserialize<T>(json);
```

Comment:
WTF?! are you telling me there's no faster way to clone an object in C# rather than serialize-deserialize?..

Reply:
Well with custom Clone or Copy constructors, but this is easier. And JSON is so well optimized these days it's fast.

Reply:
It still looks insane. It cannot be faster than doing effectively a `memcpy(from, to)`.

Reply:
`memcpy` is over simplifying it a bit, we have our own DeepCopy funtion (https://github.com/jellyfin/jellyfin/blob/master/MediaBrowser.Controller/Entities/BaseItemExtensions.cs#L63), but I'm pretty sure the json serializer is still faster last time I checked.

Reply:
I'm gonna take your word on that. That's still... weird at best, but whatever %)